src/components/Sum.jsx

```diff
@@ -1 +1,20 @@
-// export function Sum() {}
+import React from 'react';
+import Proptypes from 'prop-types';
+
+export function Sum({ a, b }) {
+  return (
+    <p>
+      {`Sum of ${a} and ${b} is ${a + b}`}
+    </p>
+  );
+}
```

Comment:
also, if you return a value immediately - you can use an arrow function 
```suggestion
export const Sum = ({ a, b }) => (
  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
)
```

Reply:
Just for your information